playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -20,6 +25,38 @@
     pod_restarts,
 )
 
+class RelatedPodParams(ActionParams):
+    """
+    :var block_type: output format type of the action.
+    """
+    output_format: str = "table"  # table, json
```

Comment:
The comment "block_type" and variable name "output_format" don't match, i would also add the types to the comment so it will automatically show up in the docs

personally since its only two types I would do something like output_as_table: bool =True but what you wrote is okay too